doc/manual/manual.tex

```diff
@@ -449,14 +449,15 @@ \subsection{Basic equations}
   \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
   \\
   &\quad
-  +\alpha T \left( \mathbf u \cdot \nabla p \right)
+  +\alpha \hat{T} \left( \mathbf u \cdot \nabla p \right)
```

Comment:
Here and three lines below, it should remain to be `T`: the form discussed here is simply the continuous differential equation. There is no time stepping yet at this point, and consequently no need to approximate T in any way.